Assets/BossRoom/Scripts/Server/Game/Entity/ServerDisplacerOnParentChange.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    /// <summary>
+    /// Component to simply play a raising/descending animation when this NetworkObject's parent NetworkObject changes.
+    /// </summary>
+    /// <remarks>
+    /// This script will likely be refactored and/or removed removed with a NetworkAnimator-based solution.
```

Comment:
current netcode version used by BR already has a NetworkAnimator, anything preventing us from using it? I thought it was just going to be a "drop on my object and it works", would that be different with newer versions?

Reply:
So initially I gave this a go with NetworkAnimator. The thing is, the animation would be applying root motion to the root pot object, which isn't supported AFAIK.

Reply:
I see. It might be worth it to add a note in the comment about root motion not being supported by NA

Reply:
Updated remarks to elaborate on the "why".